app/models/health/qualifying_activity.rb

```diff
@@ -275,7 +277,13 @@ def client_reached_title key
       client_reached[key&.to_sym].try(:[], :title) || key
     end
 
+    def from_epic?
+      source_type == 'Health::EpicQualifyingActivity'
+    end
+
     def mode_of_contact_is_other?
+      return false if from_epic?
```

Comment:
changing mode_of_contact_is_other also impacts the [display_sections method](https://github.com/greenriver/hmis-warehouse/blob/c7705c39ed69c782e2da319041112834b57a56c1/app/models/health/qualifying_activity.rb#L304) - there's also a view that seems to reference it ( app/views/health/sdh_case_management_notes/_activities.haml)

Reply:
Yes, these are also used to control visibility of the contingent data, but since, for an Epic QA, the contingent data will always be blank, I don't think that is an issue.